src/OptimismGovernor.sol

```diff
@@ -425,9 +450,9 @@ contract OptimismGovernor is
         bytes memory proposalData,
         string memory description,
         uint8 proposalType
-    ) public virtual onlyManagerOrTimelock returns (uint256 proposalId) {
+    ) public virtual onlyValidProposer returns (uint256 proposalId) {
         address proposer = _msgSender();
-        if (proposer != manager) {
+        if (proposer != manager && proposer != authorizedProposer) {
```

Comment:
If the validator bypasses the proposal threshold are there limits to users who submit a proposal that is approved?

Reply:
For a proposal to reach this stage of creating the proposal on the Governor, there needs to pass some checks, like gathering the needed approvals and creating it (calling
 `moveToVote -> proposeWithModule`) during a valid voting cycle. 
 There is no limit of voting power a user must have in order to submit specific proposal types (Governance Fund & Council Budget) to the Validator contract, however the checks mentioned above and along with some others that run on submission should cover for the "lack" of voting power.
 What kind of limits do you have in mind?